src/Bitmap.elm

```diff
@@ -43,7 +43,7 @@ forInt n =
                         ( -117, 0 )
 
                     _ ->
-                        Debug.crash ""
```

Comment:
before this `case`, it says `if n >= 0 && n <= 9 then`, so really, this `Debug.crash` can never get hit. 

`case`s have to be exhaustive for the _entire_ type. the type system isn't smart enough to realize that we're only dealing with 0-9, so the ` _ -> Debug.crash ""` keeps the type system happy by matching "all the other values", even tho you and i know there aren't any other values to deal with.

just wanted to explain what's going on here and why there was no error message.

Reply:
ok. but Debug.crash looks like it is no longer exposed. https://package.elm-lang.org/packages/elm/core/latest/Debug so  i've added a debug.log and am returning a meaningless tuple.